addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -42,6 +42,43 @@ MockServer.include({
         }
         this._widget = options.widget;
 
+        /**
+         * Generating channel_last_seen_partner_ids/mail.channel.partner based on the member field of mail.channel
+         * which mimics the behaviour of the compute of the member field.
+         */
+        const channelMembers = data && 'mail.channel.member' in data ? data['mail.channel'].records : [];
+        this.data = data; // required to use this._getUnusedID();
```

Comment:
This is part of the call to super, it shouldn't be done here.

Maybe you should call super first and then post process the data instead of pre processing it.